app/javascript/components/card-table.jsx

```diff
@@ -16,6 +16,10 @@ const CardTable = props => {
     users
   } = props;
 
+  const findCurrentUser = (userEmail, allUsers) => {
+    return allUsers.find(it => it.email === userEmail); // TO DO : rewrite after changes in back
```

Comment:
It could be fixed here, I think https://github.com/cybergizer-hq/retrospective/blob/master/app/views/boards/show.html.erb#L52
@IvanShamatov am I right?

Reply:
Seems so, I can check it

Reply:
What should be changed on the backend?